go.mod

```diff
@@ -79,36 +79,44 @@ require (
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/reform.v1 v1.5.1
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.24.8
-	k8s.io/apimachinery v0.24.8
-	k8s.io/client-go v0.24.8
+	k8s.io/api v0.25.4
```

Comment:
Let's pin it to 0.24

Reply:
Ohh that slipped through, thanks for noticing.